shanoir-uploader/src/main/java/org/shanoir/uploader/service/rest/ServiceConfiguration.java

```diff
@@ -89,12 +93,24 @@ public Boolean isProxyEnabled() {
 	}
 
 	public String getTestURL() {
-		if (ShUpConfig.proxyProperties.getProperty("proxy.test.url") != null
-				&& !ShUpConfig.proxyProperties.getProperty("proxy.test.url").equals("")) {
-			return (ShUpConfig.proxyProperties.getProperty("proxy.test.url"));
-		} else {
-			return "https://shanoir.irisa.fr";
-		}
-	}
-
+		String profile = ShUpConfig.basicProperties.getProperty("profile");
```

Comment:
Hi @LaurentPV, thank you for your PR.
What happens in case no profile is pre-configured in basicProperties? what is true in most of the cases.